lib/raix/predicate.rb

```diff
@@ -26,12 +26,9 @@ module Raix
   #   question = Question.new
   #   question.ask("Is Ruby a programming language?")
   module Predicate
+    extend ActiveSupport::Concern
     include ChatCompletion
 
-    def self.included(base)
-      base.extend(ClassMethods)
-    end
-
```

Comment:
That seems like an unnecessary change, but without it the new class level methods I added actually don't show up when you only include this module. The Rails magic strikes once more!